### PR TITLE
Revert new theme to Webpack 2

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Because webpack 4 sucks in dev mode. We must wait for webpack 5 which will include automatic css integration.
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | You should be able to use `npm run dev`, and `npm run build`, but there is a `npm run watch` alias for `dev` mode.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12739)

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

File: admin-dev/themes/new-theme/.webpack/common.js [hunk 1]
```diff
@@ -120,11 +164,11 @@ module.exports = {
     ],
   },
   plugins: [
+    new ExtractTextPlugin('theme.css'),
     new CleanWebpackPlugin(['public'], {
       root: path.resolve(__dirname, '../'),
       exclude: ['theme.rtlfix']
     }),
-    new VueLoaderPlugin(),
```
Comment (mickaelandrieu) [inline admin-dev/themes/new-theme/.webpack/common.js:127]: what is the role of the VueLoaderPlugin?

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/.webpack/common.js:127]: This one is not compatible with webpack2, there is already a rule for vue-loader

**Comment by Quetzacoalt91** [comment]
Thanks @PierreRambaud

**Comment by eternoendless** [comment]
Maybe we can try webpack 3 in the meantime?

**Comment by davidglezz** [comment]
Webpack 5 has been released!
https://webpack.js.org/blog/2020-10-10-webpack-5-release/

**Comment by PierreRambaud** [comment]
> Webpack 5 has been released!
> https://webpack.js.org/blog/2020-10-10-webpack-5-release/

Not sure talking on this old pull request is a good idea :wink: 
If you have time, could you please create an issue? :)

**Pull request merged**